lib/MusicBrainz/Server/Data/CritiqueBrainz.pm

```diff
@@ -63,7 +63,7 @@ sub _parse_review {
     return Review->new(
         id => $data->{id},
         created => DateTime->from_epoch(epoch => str2time($data->{created})),
-        body => markdown($data->{text}),
+        body => $data->{text} ? markdown($data->{text}) : '',
```

Comment:
I think it'd be good practice to use `non_empty` here to avoid problems with `'0'` (but I'm guessing there's a character minimum on reviews anyway).